docs/release-process.md

```diff
@@ -1,12 +1,49 @@
 # Release Process
 
+## Get Approval
+
+### 1. Create Release Note documentation
+
+Create a release note (using the ECP Beta Release Note template) by creating a
+new confluence page and changing the title to correspond with the release number
+under the Release Process confluence page and the page must have the following
+information e.g.
+[`Release 1`](https://dfedigital.atlassian.net/wiki/spaces/TP/pages/3021406211/Release+1)
+Information | Notes --- | --- Release Date | Planned production release date
+Approver | Name of the person who is going to approve the release Approval Date
+| The date approval was granted Status | Set the PENDING status, when the team
+is ready to release in production. Summary | Provide a short summary of the
+release and it’s associated business value Important highlights from this
+release | High level explanation of what is included in this release. NOTE: This
+section and CHANGELOG release notes must match. All update for this release |
+Provide a list of Jira tickets that are going to released with links to the Jira
+issue. Tickets should be split into the features and bugs section of the note.
+Testing | Provide the testing approach you are taking/have taken for the release
+and links to any results/reports from lower environment testing. Rollback |
+Define the rollback plan if the event that the release doesn’t go as planned.
+Failover Plan | Define the failover plan in the event of catastrophic failure.
```

Comment:
What is all this @mashakir?

Reply:
This is a description of headings for the release note page on the confluence. All information is defined on the release process confluence page.